Python/Product/Profiling/Profiling/LaunchProfiling.xaml

```diff
@@ -152,6 +166,24 @@
                 </Grid>
             </GroupBox>
         </Grid>
+
+<!--
+        <Grid Grid.Row="1">
+            <CheckBox Grid.Row="1" Margin="12 12 12 4"
+                      Content="{x:Static ui:Strings.LaunchProfiling_MixedModeProfiling}"
+                      Visibility="{Binding IsExternalProfilerEnabled}"
+                      IsEnabled="{Binding IsVTuneAvailable}"
+                      IsChecked="{Binding UseVTune}" />
+        </Grid>
+        -->
+        <Grid Grid.Row="1">
```

Comment:
That looks good to me. Please remove the commented out version.

Reply:
So sorry about this, please find it corrected in c8561d933